dashboard/__tests__/ordersViewModelsTests.ts

```diff
@@ -119,4 +135,40 @@ describe('Constructor', () => {
 		const _ = ordersViewModel.getOrders();
 		expect(mockGetOrders).toHaveBeenCalled();
 	});
+
+	test('synchronise items in model', async () => {
```

Comment:
add a check that the items are initialized correctly after "ordersViewModel.getItems()" call